method/application/get.js

```diff
@@ -0,0 +1,63 @@
+const pull = require('pull-stream')
+module.exports = function GroupApplicationGet (server) {
+  return function groupApplicationGet (applicationId, cb) {
+    const query = [
+      {
+        $filter: {
+          dest: applicationId
+        }
+      }
+    ]
+
+    server.get({ id: applicationId, private: true }, (rootErr, rootData) => {
+      if (rootErr) cb(rootErr)
+      pull(
+        server.backlinks.read({ query }),
+        pull.collect((err, data) => {
+          if (err) cb(err)
+          let final = {
+            root: applicationId,
+            applicantId: rootData.author,
+            groupId: rootData.content.groupId,
+            recps: rootData.content.recps
+          }
+          /* If there's only the root message */
+          if (!data || data.length < 1) {
+            const res = {
+              ...final,
+              ...rootData.content,
```

Comment:
I don't think `rootData.content` should be merged in, as I think the data structure should be the  same whether there's updates or not. 
I think the solution is to just remove this lin

Reply:
Ok, makes sense since there's no reason to show the tangle info here.